vnext/local-cli/runWindows/utils/msbuildtools.js

```diff
@@ -125,10 +117,7 @@ function VSWhere(requires, version, property) {
       if (toolsPathOutput) {
         let toolsPathOutputStr = toolsPathOutput[1];
         // Win10 on .NET Native uses x86 arch compiler, if using x64 Node, use x86 tools
-        if (
-          version === '15.0' ||
-          (version === '14.0' && toolsPathOutputStr.indexOf('amd64') > -1)
-        ) {
+        if (version === '16.0') {
           toolsPathOutputStr = path.resolve(toolsPathOutputStr, '..');
```

Comment:
else, break the build.
If we aren't supporting v14/15 (which I'm fine with saying we don't), then let's not put tarpit traps around it :)

Reply:
Yeah, there's a lot of cleanup in the cli to remove old versions.

Reply:
the min versions are all at 16.0 now

Reply:
will that actually prevent building? I think the version number in the proj is just a VS thing.
Consider actually breaking here or below where you check env.VisualStudioVersion if it isn't 16

---
In reply to: [401975213](https://github.com/microsoft/react-native-windows/pull/4101#discussion_r401975213) [](ancestors = 401975213)